apps/multisig-signer/src/app/components/web3-connector/web3-connector.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEnvironment } from '@vegaprotocol/environment';
+import { useEthereumConfig } from '@vegaprotocol/web3';
+import { Button, Splash, AsyncRenderer } from '@vegaprotocol/ui-toolkit';
+import { Web3ConnectDialog } from '@vegaprotocol/web3';
+import { useWeb3React } from '@web3-react/core';
+import type { ReactElement } from 'react';
+import { useEffect, useMemo } from 'react';
+import { createConnectors } from '../../lib/web3-connectors';
+
+interface Web3ConnectorProps {
+  children: ReactElement;
+  dialogOpen: boolean;
+  setDialogOpen: (open: boolean) => void;
+}
+
+export function Web3Connector({
```

Comment:
question (blocking): why not use web3-container from web3 lib? This file looks _almost_ the same. If the one in the lib isn't suitable, why? and can we update it to be better fit for purpose